version.txt

```diff
@@ -1 +1 @@
-0.1.0
\ No newline at end of file
+0.1.1
```

Comment:
Is this just a patch release?

Reply:
Oops, I believe it's a minor release since it includes minor bug fixes and features that are backwards compatible

Reply:
LGTM, please bump the minor version. Approving.